### DISCUSSION TITLE
How to implement events

### DISCUSSION THREAD

**Discussion by CurlyMoo** [CurlyMoo/rules#4] [Q&A]
I've implemented modular events. This should allow ESPEasy like events triggered by various reasons like timers or as defined as reusable functions.

@TD-er, what do you think? And for a seperate topic. What do you still miss?

**Comment by TD-er** [comment]
Please ask me again tomorrow evening, as I'm now a bit too tired and tomorrow I'll be away for a big part of the day.

**Reply by CurlyMoo** [reply]
Take your time. I can wait 🌙

**Reply by CurlyMoo** [reply]
In case you indeed need that reminder, a reminder @TD-er

**Reply by TD-er** [reply]
I was a bit too tired last night, so will have a reply this evening :)
